styles.css

```diff
@@ -44,42 +106,44 @@ body {
   padding: 3px 8px;
 }
 
+.book-info {
+  font-size: 1rem;
+  font-weight: 900;
+  color: #fff;
+}
+
+#set-date {
+  width: 80%;
+  margin: 0 auto;
+  text-align: right;
+}
+
 .remove-button {
   padding: 2px 8px;
   box-shadow: 3px 3px 2px 2px #000;
+  color: black;
 }
 
 .book-entry:nth-child(even) {
-  background-color: white;
+  background-color: rgba(255, 192, 203, 0.471);
 }
 
 .book-entry:nth-child(odd) {
-  background-color: yellowgreen;
+  background-color: rgba(0, 0, 255, 0.368);
 }
 
 #form {
-  display: flex;
+  width: 35%;
+  display: none;
   flex-direction: column;
-  align-items: flex-end;
+  align-items: center;
   text-align: center;
   gap: 2px;
 }
 
-.divide {
-  border: 2px solid #000;
-  border-radius: 50%;
-  width: 100px;
-  margin-bottom: 15px;
-}
-
-.form-heading {
-  text-align: center;
-  padding-bottom: 5px;
-  font-size: 20px;
-  font-family: sans-serif;
-}
-
 .input-area {
+  margin-block: 5px;
+  width: 100%;
   padding: 3px 20px 3px 5px;
   border: 2px solid #000;
   text-align: start;
```

Comment:
- [ ] I have noticed that the input text is having the same color as the background 😞 . This hinders the user from seeing the inputs entered. Therefore, I suggest that you change the color of the `input text` to make it visible ☺️